docs/README.rst

```diff
@@ -252,6 +254,28 @@ Some of these cert/key pairs can be wildcard (\*.mycf.com).
 
 .. _health checks:
```

Comment:
You've orphaned this section tag. Please add a new one like this for your new section, and move the health checks tag directly before the Health Checks header.